_config.yml

```diff
@@ -92,6 +93,14 @@ android-store: "Google Play"
 taet: "Telerik Application Enrollment Token"
 mvs: "Microsoft Visual Studio"
 
+# Aliases for the AB clients
+"ab-web": "in-browser client"
+"ab-win": "classic Windows desktop client"
+"ab-vse": "extension for Visual Studio"
+"ab-cli": "command-line interface"
+"ab-sp": "package for Sublime Text"
+"ab-proton": "universal desktop client"
+
```

Comment:
@Mike4o @AntonDobrev @mkapostolov Could you please share your opinion about this.